docs/modules/operation/pages/flows/thresholding.adoc

```diff
@@ -0,0 +1,74 @@
+[[ga-flow-support-data-collection]]
+= Thresholding Flow applications
+
+Flows are categorized in applications using the <<flows/classification-engine.adoc#ga-flow-support-classification-engine, Classification Engine>>.
+OpenNMS supports to sum up bytesIn/bytesOut data of flow records based on these Flow applications.
```

Comment:
OpenNMS supports summing up the bytesIn/bytesOut data of flow records based on these flow applications.